autopush/websocket.py

```diff
@@ -1480,23 +1372,15 @@ def send_notification(self, update):
         update notification from an endpoint.
 
         """
-        chid, version = (update["channelID"], update["version"])
-        if not self.ps.use_webpush and \
-           self._newer_notification_sent(chid, version):
-            return
-
-        if self.ps.use_webpush:
-            # Create the notification
-            notif = WebPushNotification.from_serialized(self.ps.uaid_obj,
-                                                        update)
-            self.ps.direct_updates[chid].append(notif)
-            if notif.topic:
-                self.metrics.increment("ua.notification.topic")
-            self.sendJSON(notif.websocket_format())
-        else:
-            self.ps.direct_updates[chid] = version
-            msg = {"messageType": "notification", "updates": [update]}
-            self.sendJSON(msg)
+        chid = update["channelID"]
+
+        # Create the notification
+        notif = WebPushNotification.from_serialized(self.ps.uaid_obj,
```

Comment:
nitpick: this and I think some other lines in this file can be unwrapped (fit into one line now)